### PR TITLE
Update bilibili-official from 1.1.3 to 1.2.1

### PR DESCRIPTION
**Pull request by i0ntempest**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by i0ntempest** [comment]
They might have changed the livecheck URL, but there's no way of knowing.

**Review by bevanjkay — DISMISSED** [review]

File: Casks/bilibili-official.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   homepage "https://app.bilibili.com/"
 
   livecheck do
-    url "http://api.bilibili.com/x/elec-frontend/update/latest-mac.yml"
+    url "http://api.bilibili.com/x/elec-frontend/update/latest.yml"
```
Comment (bevanjkay) [inline Casks/bilibili-official.rb:13]: @i0ntempest This is the feed for the windows version of the application. The mac version is still pointing to the previous version.

Comment (i0ntempest) [reply Casks/bilibili-official.rb:13]: Yes, on the official website the 2 electron versions uses the same “desktop version” change log. If this is not acceptable then I guess I can only remove the livecheck.

Comment (SMillerDev) [reply Casks/bilibili-official.rb:13]: Could it be that the macOS version isn't released yet?

Comment (i0ntempest) [reply Casks/bilibili-official.rb:13]: > Could it be that the macOS version isn't released yet?

I’m sure it is. The version downloaded from the web page is indeed 1.2.1.

Comment (SMillerDev) [reply Casks/bilibili-official.rb:13]: So can we check that webpage?

Comment (i0ntempest) [reply Casks/bilibili-official.rb:13]: https://app.bilibili.com/
it’s all in Chinese though.

Comment (SMillerDev) [reply Casks/bilibili-official.rb:13]: Right, so the download URL is on that website. We can use the `extract_plist` strategy for this.

**Review by miccal — APPROVED** [review]

**Pull request merged**